README.md

```diff
@@ -13,7 +13,41 @@ But the resources can be varied based on availability. However, make sure to
 use `--iso-url centos` part in above command as we have setup things on CentOS
 based minishift VM.
 
-Once the VM is ready, spin up a Jenkins server that can be used by the Jenkins
+Once the VM is up, log into the vm using the command `$ minishift ssh`. Once you are in, apply the following steps to install docker distribution registry. You can also install it on a seperate system, or your base machine, just ensure it is reachable from the minishift VM.
+
+    $ yum -y install docker-distribution
+
+If you are running on a separate machine, as it would be recommended, you can skip the below step of configuring the registry. Otherwise please edit the config file located at `/etc/docker-distribution/registry/config.yml` to setup ports/storage etc. 
+
+    version: 0.1
+    log:
+      fields:
+        service: registry
+    storage:
+        cache:
+            layerinfo: inmemory
+        filesystem:
+            rootdirectory: /var/lib/registry
+    http:
+        addr: :10000
```

Comment:
It doesn't matter what the port is so long as it matches accross the board. 

The reason I chose other than 5000 is because openshift has its own internal registry, and I didn't want to take chances with conflicts.